tools/check_repositories.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eu
+
+# Check whether any git repositories are defined.
+# Git repository definition contains `commit` and `remote` fields.
+if grep -Rn "commit =\|remote =" --exclude-dir=bazel-envoy --include=*.bzl .; then
```

Comment:
only grep `bazel/` and `api/bazel` directory? or use `-r` instead of `-R` then you don't need exclude-dir.